acceptance/config/aio/options.rb

```diff
@@ -1,18 +1,9 @@
 {
-  :log_level     => 'info',
-  :type          => 'packages',
-  :forge_host    => 'forge-aio01-petest.puppetlabs.com',
-  :load_path     => './lib/',
```

Comment:
i bias towards having most options in the options file rather than the rakefile so one can run from beaker stand-alone.  load_path is one that is required in most cases. :forge_host and  :ssh => :keys are others